src/Backend/Modules/Faq/Actions/Edit.php

```diff
@@ -108,12 +108,13 @@ private function validateForm(): void
                 // build item
                 $item = [];
                 $item['id'] = $this->id;
-                $item['meta_id'] = $this->meta->save(true);
+                $item['meta'] = $this->meta;
                 $item['category_id'] = $this->form->getField('category_id')->getValue();
                 $item['language'] = $this->record['language'];
                 $item['question'] = $this->form->getField('title')->getValue();
                 $item['answer'] = $this->form->getField('answer')->getValue(true);
                 $item['hidden'] = $this->form->getField('hidden')->getValue();
+                $item['sequence'] = $this->record['sequence'];
```

Comment:
Don't change anything in the actions, only in the model.

Reply:
This is needed to make the update work, otherwise the sequence will be lost

Reply:
And how did it work before?

Reply:
Only the fields given in the array would be updated (in sequence only sequence was given for example).

Reply:
Fixed 232348709